cafeteria/src/main/java/org/itt/utility/Client.java

```diff
@@ -0,0 +1,42 @@
+package org.itt.utility;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.net.Socket;
+
+public class Client {
+    private static ObjectOutputStream outputStream;
+    private static ObjectInputStream inputStream;
+    private static Socket socket;
+
+    public static void main(String[] args) {
+        try {
+            socket = new Socket("localhost", 8080);
```

Comment:
Let's not hardcode here. It is better to wrap this in a function and pass as parameters.